### PR TITLE
Create Henry_Huang.markdown

### PR DESCRIPTION
**Pull request by hen-w**

### CODE REVIEW

File: _people/Henry_Huang.markdown [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+address: 619 SSB
+specialties: Numerical Relativity
+---
+
+##### Research Projects
+
+Ccz4 with FD-DG hybrid scheme
+
+##### Biography
+
```
Comment (nilsdeppe) [inline _people/Henry_Huang.markdown:19]: Can you fill this out a bit more?

File: _people/Henry_Huang.markdown [hunk 2]
```diff
@@ -0,0 +1,20 @@
+---
```
Comment (nilsdeppe) [inline _people/Henry_Huang.markdown:1]: Can you also add a headshot in `images/people/Henry_Huang.jpg`

File: _people/Henry_Huang.markdown (hunk 1 above)
Comment (hen-w) [reply _people/Henry_Huang.markdown:19]: Done @nilsdeppe

File: _people/Henry_Huang.markdown (hunk 2 above)
Comment (hen-w) [reply _people/Henry_Huang.markdown:1]: Done @nilsdeppe

**Review by nilsdeppe — APPROVED** [review]

**Pull request merged**